freeipa/src/main/java/com/sequenceiq/freeipa/sync/ProviderChecker.java

```diff
@@ -91,21 +101,25 @@ private InstanceStatus updateStatuses(CloudVmInstanceStatus vmInstanceStatus, In
             default:
                 LOGGER.info(":::Auto sync::: the '{}' status is not converted", vmInstanceStatus.getStatus());
         }
-        if (updateStatus) {
-            instanceMetaDataService.save(instanceMetaData);
+        if (!flowLogService.isOtherFlowRunning(stack.getId())) {
+            if (updateStatus) {
+                instanceMetaDataService.save(instanceMetaData);
+            } else {
+                LOGGER.debug("updateStatus flag is false, don't update status");
+            }
+        } else {
+            LOGGER.debug(":::Auto sync::: do not update instance status to {} because flow is running for stack: {}", status, stack.getName());
         }
         return status;
     }
 
     private void setStatusIfNotTheSame(InstanceMetaData instanceMetaData, InstanceStatus newStatus) {
-        if (instanceMetaData.getInstanceStatus() != newStatus) {
-            if (updateStatus) {
-                instanceMetaData.setInstanceStatus(newStatus);
-                LOGGER.info(":::Auto sync::: The instance status updated from {} to {}", instanceMetaData.getInstanceStatus(), newStatus);
-            } else {
-                LOGGER.info(":::Auto sync::: The instance status would be had to update from {} to {}",
-                        instanceMetaData.getInstanceStatus(), newStatus);
-            }
+        if (updateStatus) {
```

Comment:
this check has been removed: `instanceMetaData.getInstanceStatus() != newStatus`
is it on purpose? I mean we will log here even no change happens in this case

Reply:
thanks! it was a mistake :)